.circleci/config.yml

```diff
@@ -34,6 +34,15 @@ e2eTest: &e2eTest
             -w /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
             golang:1 go test -v -tags k8srequired ./${E2E_TEST_DIR}/
 
+    - run:
+        name: Export logs
```

Comment:
Probably this should be run always (especially on failure), like cleanup command bellow.

Reply:
Yes good call, thx!